android/build.gradle

```diff
@@ -26,6 +26,14 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+    packagingOptions {
+        pickFirst 'lib/x86/libc++_shared.so'
```

Comment:
According to https://developer.android.com/reference/tools/gradle-api/7.4/com/android/build/api/dsl/PackagingOptions#pickFirst(kotlin.String) it is deprecated now. Also any chance to investigate why it doesn't work for you if you do it in the host app?